mvpa2/misc/neighborhood.py

```diff
@@ -253,10 +256,11 @@ def __init__(self, radius, inner_radius, **kwargs):
                   "than the radius (got %g)" % (inner_radius, radius)
         Sphere.__init__(self, radius, **kwargs)
         self._inner_radius = inner_radius
+        self.include_center = include_center
 
     def __repr__(self, prefixes=[]):
         return super(HollowSphere, self).__repr__(
-            ['inner_radius=%r' % (self._inner_radius,)])
+            ['inner_radius=%r' % (self._inner_radius,),'include_center=%r' % (self.include_center,)])
```

Comment:
you did it almost right ;-)  so I will merge it and tune it up ... if you use

```Python
+ _repr_attrs(self, ['include_center'], default=False))
```
then it wouldn't even list if it it matches the default